templates/dependapanda.text.erb

```diff
@@ -1,9 +1,15 @@
-You have <%= @message.reduce(0) { |acc, pr| acc + pr.fetch(:pr_count) } %> Dependabot PRs open on the following apps:
+<%= prs_count_message %>
 
-<% @message.each do |pr| -%>
-<<%= pr[:repo_url] %>|<%= html_encode(pr[:repo_name]) %>> (<%= pr[:pr_count] %>, <% if pr[:pr_count] > 1 -%>
-oldest: <%= days_plural(pr[:oldest_pr])%>, newest: <%= days_plural(pr[:newest_pr]) %>)
-<% else -%>
-opened <%= days_plural(pr[:oldest_pr]) %>)
+<% @repos.each do |repo| -%>
+<<%= repo[:repo_url] %>|<%= html_encode(repo[:repo_name]) %>> (<%= repo[:pr_count] %>, <%= repo_details(repo) %>)
+<% end -%>
+
+<% if @team.security_alerts %>
+<%= security_alerts_intro %>
+
+<% if @security_prs_ordered.any? -%>
```

Comment:
You can remove this `if` - I think `security_prs_ordered` defaults to an empty array, so the `each` loop below would just be skipped over.